pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -102,3 +104,13 @@ func (s *ObjectStoreSpec) GetServiceServingCert() string {
 func (c *CephObjectStore) GetStatusConditions() *[]Condition {
 	return &c.Status.Conditions
 }
+
+// IsRGWSSEKMSEnabled return SSE-KMS is configured for RGW
+func (s *ObjectStoreSpec) IsRGWSSEKMSEnabled() bool {
+	return len(s.AWSServerSideEncryptionType) == 0 || strings.Contains(s.AWSServerSideEncryptionType, "kms")
+}
+
+// IsRGWSSES3Enabled return SSE-S3 is configured for RGW
+func (s *ObjectStoreSpec) IsRGWSSES3Enabled() bool {
+	return len(s.AWSServerSideEncryptionType) > 0 && strings.Contains(s.AWSServerSideEncryptionType, "s3")
```

Comment:
Why is the length test different than `IsRGWSSEKMSEnabled`?

Reply:
RGW will be configured with KMS settings if this field is not set(to resolve with back compactablity issue)